fre/yamltools/combine_yamls.py

```diff
@@ -369,12 +366,12 @@ def _consolidate_yamls(yamlfile,experiment,platform,target,use):
         raise ValueError("'use' value is not valid; must be 'compile' or 'pp'")
 
 @click.command()
-def consolidate_yamls(yamlfile,experiment,platform,target,use):
+def _consolidate_yamls(yamlfile,experiment,platform,target,use):
```

Comment:
Okay, so the plain name is able to be called from other methods (pythonically), and the underscore methods are used for the click interface?

Reply:
@cwhitlock-NOAA I thought the underscore method was something for the pp wrapper?

Reply:
> Okay, so the plain name is able to be called from other methods (pythonically), and the underscore methods are used for the click interface?

CORRECT!



> @cwhitlock-NOAA I thought the underscore method was something for the pp wrapper?

and also, let me clear, @cwhitlock-NOAA introduced me to the idea. I think it just got used backwards in a spot or two. but conventionally to `pylint`, `_some_function_name` is the "protected" one... in our case, it's "protected" for `click`.